java-backend/src/main/resources/org/kframework/backend/java/symbolic/hooks.properties

```diff
@@ -157,7 +157,7 @@ STRING.floatFormat : org.kframework.backend.java.builtins.BuiltinStringOperation
 FLOAT.int2float : org.kframework.backend.java.builtins.BuiltinFloatOperations.int2float
 FLOAT.float2int : org.kframework.backend.java.builtins.BuiltinFloatOperations.float2int
 STRING.token2string : org.kframework.backend.java.builtins.BuiltinStringOperations.token2string
-STRING.string2token : org.kframework.backend.java.builtins.BuiltinStringOperations.string2token
+STRING.parseToken : org.kframework.backend.java.builtins.BuiltinStringOperations.string2token
```

Comment:
Should the builtin operator be renamed also? Or duplicated and keep `parseToken` as an alias? Not really worth looking into, just a thought.